python/cuml/preprocessing/LabelEncoder.py

```diff
@@ -55,71 +55,55 @@ class LabelEncoder(Base):
 
     .. code-block:: python
 
+        import cudf
+        import dask_cudf
         from cudf import DataFrame, Series
+        from cuml.dask.preprocessing.LabelEncoder import LabelEncoder
+        from dask.distributed import Client, LocalCluster
 
-        data = DataFrame({'category': ['a', 'b', 'c', 'd']})
+        # create a local cluster with 4 workers
+        n_workers = 2
+        cluster = LocalCluster(n_workers=n_workers)
+        client = Client(cluster)
+
+        data = dask_cudf.from_cudf(DataFrame({'category': ['a', 'b', 'c', 'd']}),npartitions=2)
 
         # There are two functionally equivalent ways to do this
         le = LabelEncoder()
         le.fit(data.category)  # le = le.fit(data.category) also works
         encoded = le.transform(data.category)
-
         print(encoded)
 
         # This method is preferred
         le = LabelEncoder()
         encoded = le.fit_transform(data.category)
-
         print(encoded)
 
         # We can assign this to a new column
-        data = data.assign(encoded=encoded)
-        print(data.head())
+        data = data.assign(encoded=encoded.values) # NOTE - This uses .values different from cudf assign
+        print(data.head()) 
 
         # We can also encode more data
-        test_data = Series(['c', 'a'])
+        test_data = dask_cudf.from_cudf(Series(['c', 'a']), npartitions=2)
         encoded = le.transform(test_data)
-        print(encoded)
 
         # After train, ordinal label can be inverse_transform() back to
         # string labels
         ord_label = cudf.Series([0, 0, 1, 2, 1])
-        ord_label = dask_cudf.from_cudf(data, npartitions=2)
+        ord_label = dask_cudf.from_cudf(ord_label, npartitions=2)
```

Comment:
Why are you changing the example of the single GPU LabelEncoder to use dask_cudf? That corresponds to tge cuml.dask implementation: https://github.com/rapidsai/cuml/blob/branch-21.10/python/cuml/dask/preprocessing/LabelEncoder.py

Reply:
Modified this example for use with the single GPU LabelEncoder as originally written. Please verify that the output of this change provides the expected result.